assets/src/edit-story/components/inspector/prepublish/prepublishChecklistProvider.js

```diff
@@ -47,46 +53,87 @@ function PrepublishChecklistProvider({ children }) {
     return { ...story, pages };
   });
 
+  const isStoryLoaded = story?.pages.length > 0;
```

Comment:
Initially when we're looking at checklist values the story hasn't been received yet which gives false positives, by checking for pages we can make sure we have a story present to run checks on.